### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bscheshirwork**
"Usage" chapter is not contain full example of common usage case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,35 @@ to the `require` section of your `composer.json`.
 Usage
 -----
 
+Configure socialNetworks in config file
+
+```php
+    'components' => [
```
Comment (greeflas) [inline README.md:70]: Can you remove this extra `Tab`, please?

File: README.md [hunk 2]
```diff
@@ -64,6 +64,35 @@ to the `require` section of your `composer.json`.
...
+    // ...
```
Comment (greeflas) [inline README.md]: Let's place this comment on the same `Tab` level as `socialShare` key

File: README.md [hunk 3]
```diff
@@ -64,6 +64,35 @@ to the `require` section of your `composer.json`.
...
+                'vkontakte' => [
+                    'class' => \ymaker\social\share\drivers\Vkontakte::class,
+                ],
+                'facebook' => [
+                    'class' => \ymaker\social\share\drivers\Facebook::class,
+                ],
+                'odnoklasniki' => [
+                    'class' => \ymaker\social\share\drivers\Odnoklassniki::class,
+                ],
+                'viber' => [
```
Comment (greeflas) [inline README.md]: Can you remove `viber` and `whatsapp`? I think 3 social networks will be enough.

**Review by greeflas — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (bscheshirwork) [reply README.md]: Ok.

File: README.md (hunk 1 above)
Comment (bscheshirwork) [reply README.md:70]: This is ordinary lvl of tab in main config. I think this is easily to include to config.

Comment (bscheshirwork) [reply README.md:70]: @greeflas your opinion? We need to remove tab anyway?

Comment (greeflas) [reply README.md:70]: @bscheshirwork oh, I see what you mean. Let's keep it.

**Comment by greeflas** [comment]
@bscheshirwork Merged. Thank you a lot!

**Pull request merged**